UI/Utils/WebDriverExtension.cs

```diff
@@ -0,0 +1,74 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using UI.WebDriver;
+using System;
+
+namespace UI.Utils
+{
+    public static class WebDriverExtension
+    {
+        private static void WaitElementIsVisible(By locator, int timeSeconds)
+        {
+            WebDriverWait wait = new WebDriverWait(Browser.GetDriver(), TimeSpan.FromSeconds(timeSeconds));
+            wait.Until(driver => driver.FindElement(locator).Displayed);
+        }
+        public static void ClickOnElement(By locator, int waitSeconds)
+        {
+            WaitElementIsVisible(locator, waitSeconds);
+            Browser.GetDriver().FindElement(locator).Click();
+        }
+
+        public static void ClickOnElement(By locator)
+        {
+            WaitElementIsVisible(locator, 5);
+            Browser.GetDriver().FindElement(locator).Click();
+        }
+
+        public static void SendKeysToElement(By locator, string text)
```

Comment:
Let's create all functions from that file with timeout parameter like on 10 line